NOTICE

```diff
@@ -1,85 +1,6 @@
 Swan
 Copyright 2016-2017 Intel Corporation
 
-This software contains portions of pkg/errors from https://github.com/pkg/errors.
-Copyright (c) 2015, Dave Cheney <dave@cheney.net>
-All rights reserved.
+This software contains portions of kubernetes/kubernetes from https://github.com/kubernetes/kubernetes,
```

Comment:
@squall0gd - for some reason you want to replace `NOTICE` with outdated version. Fix it, please.